en/docs/deploy/migrate/prepare-for-migration.md

```diff
@@ -27,6 +27,25 @@ Before you start the migration, see the instructions given here.
     create_admin_account = false 
     ```
 
+!!! note
+   * If you are migrating from a version below IS 5.9.0 with a **JDBC primary userstore** configured, use the following configuration to disable the use of **Unique ID Userstore Managers** during the migration.
+     ```toml
+     [user_store]
+     type = "database"
+     ```
+     As the former (non-unique ID) userstore managers are no longer supported, this configuration **must** be changed to the Unique ID Userstore Manager after the migration is complete to ensure proper functionality in IS 6.0.0 by using the configuration below.
+     ```toml
+     [user_store]
+     type = "database_unique_id"
+     ```
+   * If you are migrating from a version below IS 5.10.0, make sure to disable the Groups and Roles Separation feature during the migration.
```

Comment:
This piece of instruction looks like a repetition or a conflict with the next topic on `Prepare for Groups and Roles separation`. 

Shall we have separate sub topics here as follows:

- Prepare for user store manger conflicts
- Prepare for groups and roles separation
- Prepare for Zerro down time